src/app-bundles/routes-bundle.js

```diff
@@ -15,41 +15,39 @@ import AllLocations from "../app-pages/reports/corp-office/office-location/all-l
 import DataResources from "../app-pages/data-resources/DataResourcesPage";
 import NotFound from "../app-pages/NotFound";
 
+// get the base url for which all the routes will be nested under
+const homepage = new URL(pkg.homepage);
+const baseUrl = homepage.pathname;
+
 export const RoutePaths = {
-  Home: `/`,
-  Help: `/help`,
+  Home: baseUrl,
+  Help: `${baseUrl}/help`,
   Locations: `/locations/:locationId`,
```

Comment:
Next question: is there a reason Locations is the only route that doesn't use ${baseUrl}? Or was this just an oversight?

Reply:
haha woops this was an oversight. I'll push up a fix